judge/models/problem.py

```diff
@@ -172,6 +172,9 @@ class Problem(models.Model):
                                            help_text=_('If private, only these organizations may see the problem.'))
     is_organization_private = models.BooleanField(verbose_name=_('private to organizations'), default=False)
 
+    suggesters = models.ManyToManyField(Profile, verbose_name=_('Suggesters'), blank=True, related_name="suggester",
```

Comment:
I think each problem should have only one suggester. You should use `ForeignKey(Profile, blank=True, null=True, related_name='suggested_problems', on_delete=SET_NULL)`